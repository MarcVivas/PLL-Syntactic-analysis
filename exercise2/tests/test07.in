// Expected 5
a = 5;

// Expected 3
b = 3;

// Expected -6
c = ~a;

// Expected 1
d = a & b;

// Expected 7
e = a | b;

// Expected 6
f = a ^ b;

// Expected 20
g = a << 2;

// Expected 2
h = a >> 1;

// Expected 4
i = ~b & a;

// Expected 6
j = e ^ d;

// Expected -1
k = c >> 3;

// Expected 3
l = d << 1 | b;
