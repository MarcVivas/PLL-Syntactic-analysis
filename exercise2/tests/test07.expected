a = 5
b = 3
c = -6
d = 1
e = 7
f = 6
g = 20
h = 2
i = 4
j = 6
k = -1
l = 3